docs/admin-sdk/docs/guide/2_api-reference/window.md

```diff
@@ -65,3 +65,34 @@ No parameters required.
 
 #### Return value:
 Returns a promise without data.
+
+### Get an unique identifier for the window
+
+> Available since Shopware v6.7.1.0
```

Comment:
Is 6.7.0.0 already feature complete? :thinking:

Reply:
Almost. I would say we merge the documentation with v.6.7.1.0 information. If we can get this feature merged before than it is easy to change the version back to 6.7. Other way around is difficult